src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts

```diff
@@ -0,0 +1,345 @@
+import type { PropertyValues } from "lit";
+import { css, html, LitElement, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { isComponentLoaded } from "../../common/config/is_component_loaded";
+import { fireEvent } from "../../common/dom/fire_event";
+import { computeDomain } from "../../common/entity/compute_domain";
+import "../../components/ha-circular-progress";
+import {
+  createAssistPipeline,
+  listAssistPipelines,
+} from "../../data/assist_pipeline";
+import type { AssistSatelliteConfiguration } from "../../data/assist_satellite";
+import { createConfigFlow, handleConfigFlowStep } from "../../data/config_flow";
+import type { EntityRegistryDisplayEntry } from "../../data/entity_registry";
+import {
+  fetchHassioAddonsInfo,
+  installHassioAddon,
+  startHassioAddon,
+} from "../../data/hassio/addon";
+import { listSTTEngines } from "../../data/stt";
+import { listTTSEngines, listTTSVoices } from "../../data/tts";
+import type { HomeAssistant } from "../../types";
+import { documentationUrl } from "../../util/documentation-url";
+import { AssistantSetupStyles } from "./styles";
+import { STEP } from "./voice-assistant-setup-dialog";
+
+@customElement("ha-voice-assistant-setup-step-local")
+export class HaVoiceAssistantSetupStepLocal extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false })
+  public assistConfiguration?: AssistSatelliteConfiguration;
+
+  @state() private _state: "INSTALLING" | "NOT_SUPPORTED" | "ERROR" | "INTRO" =
+    "INTRO";
+
+  @state() private _detailState?: string;
+
+  @state() private _localTts?: EntityRegistryDisplayEntry[];
+
+  @state() private _localStt?: EntityRegistryDisplayEntry[];
+
+  protected override render() {
+    return html`<div class="content">
+      ${this._state === "INSTALLING"
+        ? html`<img src="/static/images/voice-assistant/update.png" />
+            <h1>Installing add-ons</h1>
+            <p>
+              The Whisper and Piper add-ons are being installed and configured.
+            </p>
+            <ha-circular-progress indeterminate></ha-circular-progress>
+            <p>
+              ${this._detailState || "Installation can take several minutes"}
+            </p>`
+        : this._state === "ERROR"
+          ? html` <img src="/static/images/voice-assistant/error.png" />
+              <h1>Failed to install add-ons</h1>
+              <p>
+                We could not automatically install a local TTS and STT provider
+                for you. Read the documentation to learn how to install them.
+              </p>
+
+              <a
+                href=${documentationUrl(
+                  this.hass,
+                  "/voice_control/voice_remote_local_assistant/"
+                )}
+                target="_blank"
+                rel="noreferrer noopenner"
+              >
+                <ha-button @click=${this._skip}>
```

Comment:
```suggestion
                <ha-button>
```

Why would I skip when I want to learn more?

Reply:
Because there is no other way to skip, and nothing else to do

Reply:
There is the skip button on the top right...

Reply:
Sorry I looked on the wrong view

Reply:
I changed it now, added back button when failed.